specification/resources/resource-manager/Microsoft.Resources/stable/2019-08-01/resources.json

```diff
@@ -4929,6 +4745,16 @@
       "type": "string",
       "description": "The ID of the target subscription."
     },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the resource group (case-insensitive).",
+      "pattern": "^[-\\w\\._\\(\\)]+$",
+      "minLength": 1,
+      "maxLength": 90
```

Comment:
need add "x-ms-parameter-location": "method" 

Otherwise Java SDK failed to generate. See https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md